### DISCUSSION TITLE
Can there be a way to make routes look like Route::get(), Route::post etc.

### DISCUSSION THREAD

**Discussion by otengkwame** [clue/framework-x#211] [Ideas]
I once tried creating a class to use this style of routing which looks like a class calling it's static methods just as we see in other frameworks but it seems my level of understanding could not help me do that.

An example

```php

Route::get('/', function () {
    return React\Http\Message\Response::plaintext(
        "Hello wörld!\n"
    );
});

```

Is there a way someone can assist me in achieving this? Or is there an advice to stop me from doing things in this manner?

**Comment by SimonFrings** [comment]
@otengkwame Thanks for bringing this up :+1: 

I'm interested in what will be achieved by doing this? There are actually a few consequences changing our current behavior to support your suggested example (can't define interfaces etc.).

In my honest opinion, I don't see anything positive gained by this other than using a similar syntax like other frameworks. What do you think?

**Comment by otengkwame** [comment]
It shouldn't be how the framework actually works, but I think I wanted a syntax like that. Looking at it, I think it is just syntactic sugar. I will figure out a way to make it work, that is if I can. It is a personal request, If you can help me
